pyqtgraph/configfile.py

```diff
@@ -57,7 +56,7 @@ def readConfigFile(fname):
     try:
         #os.chdir(newDir)  ## bad.
         with open(fname) as fd:
-            s = asUnicode(fd.read())
+            s = str(fd.read())
```

Comment:
do we need to specify `str` here, we may not if we can modify the context manager to open with text mode.

```python
with open(fname, 'rt') as fd:
   s = fd.read()
```